docs/01-about/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: What is Nordcraft?
-description: Welcome to the Nordcraft documentation. Learn how to build powerful web applications with Nordcraft through step-by-step guides and practical examples. Explore components, styling, APIs, and advanced features.
+description: Nodcraft documentation. Learn to build powerful web applications with Nordcraft through guides and examples. Explore components, styling, APIs, and advanced features.
```

Comment:
```suggestion
description: Learn to build powerful web applications with Nordcraft through guides and examples. Explore components, styling, APIs, and advanced features.
```

I don't think we need the title in the actual description so we can remove this ☺️